packages/@adobe/spectrum-css-temp/components/rule/skin.css

```diff
@@ -21,3 +21,12 @@ governing permissions and limitations under the License.
 .spectrum-Rule--small {
   background-color: var(--spectrum-rule-small-background-color);
 }
+
+@media (forced-colors: active) {
+  .spectrum-Rule {
+    forced-color-adjust: none;
```

Comment:
I don't see this doing anything? at least, not in chrome's emulation of forced colors

Reply:
It should render the rule. Without `forced-color-adjust: none;`, the background-color will be ignored and the rule renders as transparent.